test/mcmc/Inference.jl

```diff
@@ -116,19 +116,16 @@ using Turing
     @testset "Prior" begin
         N = 10_000
 
-        # Note that all chains contain 3 values per sample: 2 variables + log probability
         @testset "Single-threaded vanilla" begin
             chains = sample(StableRNG(seed), gdemo_d(), Prior(), N)
             @test chains isa MCMCChains.Chains
-            @test size(chains) == (N, 3, 1)
```

Comment:
Why remove this? Same for similar lines below.

Reply:
I'm not very bothered by this, and we can remove this line. See Slack for more comments.